kubemarine/apt.py

```diff
@@ -47,7 +48,7 @@ def create_repo_file(group, repo_data, repo_file):
     if isinstance(repo_data, list):
         repo_data_str = "\n".join(repo_data) + "\n"
     else:
-        repo_data_str = str(repo_data)
+        repo_data_str = utils.read_external(repo_data)
```

Comment:
This is an incompatibly change. Previously yum considered string as path to file with repositories content (see yum.py), and apt considered as the repositories content itself. Now both consider the string as path to file.

Reply:
Pls add **Breaking change:** section in PR

Reply:
Done